Libraries/LibWeb/HTML/Navigator.idl

```diff
@@ -74,3 +75,4 @@ Navigator includes NavigatorConcurrentHardware;
 Navigator includes NavigatorAutomationInformation;
 Navigator includes NavigatorStorage;
 Navigator includes NavigatorDeviceMemory;
+Navigator includes NavigatorCredentialsContainer;
```

Comment:
Make sure to configure your editor to leave a newline at EOF

Reply:
I am the one who complains about this at work, so much shame on myself :laughing: